planner/types/Resource.ts

```diff
@@ -2,16 +2,20 @@ import {Role} from "./Role";
 
 export class Resource {
     name: string;
-    role: Role;
+    roles: Role[];
     capacity: number;
 
-    public constructor(name: string, role: Role, capacity: number) {
+    public constructor(name: string, roles: Role[] = [], capacity: number) {
         this.name = name;
-        this.role = role;
+        this.roles = roles;
         this.capacity = capacity;
     }
 
     public satisfies(role: Role | null, NoP: number): boolean {
-        return role === this.role && NoP <= this.capacity;
+        if (!role) {
```

Comment:
take capacity into account though? or we should define completely other behavior

Reply:
Behavior will be, that Anyone can do it without assigning a resource (or assign new Anyone resource